src/webui/www/private/confirmruleclear.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="${LANG}">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>QBT_TR(Clear downloaded episodes)QBT_TR[CONTEXT=AutomatedRssDownloader]</title>
+    <link rel="stylesheet" href="css/style.css?v=${CACHEID}" type="text/css" />
+    <script src="scripts/lib/mootools-1.2-core-yc.js"></script>
+    <script src="scripts/lib/mootools-1.2-more.js"></script>
+    <script>
+        'use strict';
+
+        window.addEvent('domready', function() {
+            const rules = decodeURIComponent(new URI().getData("rules")).split('|');
+
+            $('cancelBtn').focus();
+            $('cancelBtn').addEvent('click', function(e) {
+                new Event(e).stop();
+                window.parent.MochaUI.closeWindow(window.parent.$('clearRulesPage'));
+            });
+            $('confirmBtn').addEvent('click', function(e) {
+                new Event(e).stop();
+                let completionCount = 0;
+                rules.forEach(rule => {
+                    window.parent.qBittorrent.RssDownloader.modifyRuleState(rule, "previouslyMatchedEpisodes", [], () => {
```

Comment:
Alas, we can't yet use arrow functions while we support IE11. I'd love to drop IE11 support (as Bootstrap 5 has [now done](https://twitter.com/getbootstrap/status/1245090015597580288)), but we've collectively decided to support it for a bit longer.

Reply:
As far as I could tell IE11 support had already (unintentionally) been dropped. Since the refactor from 4.2.0 IE doesn't load the page correctly.

If you still want I can replace all arrow functions.

Reply:
+1 for officially dropping support for IE 11. I think it has been long enough.